modules/end-user-guide/nav.adoc

```diff
@@ -21,7 +21,7 @@
 ** xref:importing-kubernetes-applications-into-a-workspace.adoc[]
 ** xref:remotely-accessing-workspaces.adoc[]
 ** xref:mounting-a-secret-as-a-file-or-an-environment-variable-into-a-workspace-container.adoc[]
-** xref:authentication-against-bitbucket-server-with-the-personal-access-token.adoc[]
+** xref:authentication-against-bitbucket-or-gitlab-server-with-the-personal-access-token.adoc[]
```

Comment:
authentication-against-bitbucket-or-gitlab-server-with-the-personal-access-token.adoc->authentication-against-scm-server-with-the-personal-access-token.adoc